apis/deployer/helm/types_provider.go

```diff
@@ -53,6 +62,17 @@ const (
 	UpdateStrategyPatch  UpdateStrategy = "patch"
 )
 
+// HealthChecksConfiguration contains the condiguration for health checks.
+type HealthChecksConfiguration struct {
+	// Disable allows to disable health checks.
+	// +optional
+	Disable bool `json:"disable,omitempty"`
```

Comment:
maybe lets name it default health checks => `DisableDefault`
Because we want to add custom health checks in the future.
As these custom health checks need to be configured per resource, the disable option should only disable health checks for known objects

Reply:
done!